RosNet/Exceptions/RosBagException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace RosNet;
+
+/// <summary>
+/// Exception for when RosBag is corrupt or missing fields
+/// </summary>
+public class RosBagException : Exception
+{
+    public RosBagException(string message) : base(message) { }
+}
```

Comment:
```suggestion
namespace RosNet;

/// <summary>
/// Exception for when RosBag is corrupt or missing fields
/// </summary>
public class RosBagException : Exception
{
    public RosBagException(string message) : base(message) { }
    public RosBagException(string message, Exception inner) : base(message, inner) { }
}
```
It can be helpful under e.g. debugging to include the original exception, in the case that you are re-throwing. Also: this file does not need it's own folder